views/contact.ejs

```diff
@@ -4,6 +4,7 @@
     <main class="mdl-layout__content">
         <div class="mdl-typography--text-center clemson-esports">
             <h1>Contact Us</h1>
+            <iframe src="https://docs.google.com/forms/d/e/1FAIpQLSfxFgY6umfhi8tUG-4xxVykFsRBZB2J6OLxmEeQgC-bzGxvJg/viewform?embedded=true" width="640" height="1200" frameborder="0" marginheight="0" marginwidth="0">Loading…</iframe>
```

Comment:
Would be nice to change to dark colors but this is a good starting point for us.